### PR TITLE
Add persistent notch display mode

### PR DESCRIPTION
**Pull request by panyann**
Option is enabled (not greyed out) in Reader settings only when a mobile supports notch. Default option is to NOT use the notch area on horizontal view. Also added polish translation to already fixed polish translation file.
Tested with Android developer settings.

### CODE REVIEW

**Comment by Jays2Kings** [comment]
Rather than make a new setting for something that's only for notches and only in landscape, it's fine to make it default for landscape. This seems like a change for the better for notched phones in landscape anyway

**Comment by panyann** [comment]
Changed to make it default, removed the option.

File: app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt [hunk 1]
```diff
@@ -692,6 +685,8 @@ class ReaderActivity : BaseRxActivity<ReaderPresenter>(),
 
             subscriptions += preferences.colorFilterMode().asObservable()
                 .subscribe { setColorFilter(preferences.colorFilter().getOrDefault()) }
+
+            this.setNotchCutoutMode()
```
Comment (Jays2Kings) [inline app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt]: It's better to move this into the onCreate of the reader activity

File: app/src/main/java/eu/kanade/tachiyomi/data/preference/PreferenceKeys.kt [hunk 2]
```diff
@@ -19,6 +19,8 @@ object PreferenceKeys {
 
     const val fullscreen = "fullscreen"
 
+    const val isDefaultNotchMode = "pref_notch_mode"
+
```
Comment (Jays2Kings) [inline app/src/main/java/eu/kanade/tachiyomi/data/preference/PreferenceKeys.kt]: I think you forgot to remove this one

**Comment by panyann** [comment]
Good that you stay observant.

**Comment by joseph619** [comment]
> Good that you stay observant.

Did you make the changes Jay asked for?

**Comment by panyann** [comment]
> Did you make the changes Jay asked for?

Yes, you can see the commit above my previous comment 😃

**Pull request merged**